test/TestProjects/MgmtRenameRules/MgmtRenameRules.json

```diff
@@ -0,0 +1,11761 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "ComputeManagementClient",
+        "description": "The Compute Management Client.",
+        "version": "2020-06-01"
+    },
+    "host": "management.azure.com",
```

Comment:
Please live only the parts are essential to the test, and remove the rest.

Reply:
I made a shim on this test project, removed a lot of things that are irrelevant to the renaming scenarios.